frontend/src/main/scala/bloop/data/LoadedProject.scala

```diff
@@ -0,0 +1,30 @@
+package bloop.data
+
+sealed trait LoadedProject {
```

Comment:
@tgodzik This is a new key abstraction I have added to our build load infrastructure so that we can distinguish internally whether a project has been configured or it hasn't. Even if semanticdb is not supported in a project, we still consider it configured because we always add the range positions, so in `checkForChange` we detect configured projects that don't have semanticdb enabled and we retry the resolution when `newSettings` are passed.